src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -963,6 +957,29 @@ def ensure_flow_is_serializable(self):
             self._tools_manager.wrap_tool(node.name, wrapper=_ensure_node_result_is_serializable)
 
 
+def load_worker_count_in_env(DEFAULT_WORKER_COUNT):
```

Comment:
Shall we move this method to _line_execution_process_pool.py as it seems that the _worker_count is not used directly in flow_executor?

Reply:
fixed